Editor/Manifest/Manifest.cs

```diff
@@ -246,6 +246,18 @@ public void SetDependency(string name, string version)
             var dependency = new Dependency(name, version);
             m_Dependencies[dependency.Name] = dependency;
         }
+        
+        /// <summary>
+        /// Remove <see cref="Dependency"/> by given name (if manifest contains it)
+        /// </summary>
+        /// <param name="name">Dependency name.</param>
+        public void RemoveDependency(string name)
```

Comment:
What do you think about adding `RemoveDependency` method overloading with `StansAssets.Foundation.Editor.Dependency` parameter as well?

Reply:
In my case, I don't need such method, but it sounds logical: Added.